_posts/2022-03-16-cockpit-265.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Cockpit 265
+author: skobyda
+date: '2022-03-16'
+tags: cockpit, machines
+slug: cockpit-265
+category: release
+summary: 'Animated lists, x-forwarder-for support, crypto policies, disk serial number, metrics showing busiest CPU core, manifest overrides location'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 265 and cockpit-machines 264:
+
+
+## Animate new rows in lists
+
+When new items appear on a page, they are highlighted for a short while to spot them more easily.
+
+![Peek 2022-03-15 07-47]({{ site.baseurl }}/images/265-animate-new-rows-in-lists.gif)
+
+## Support for `X-Forwarded-For`
+
+When run in a configuration involving a reverse proxy server, Cockpit can now report the client IP address as provided to it by the proxy server, rather than using the IP of the proxy server itself.  This impacts what appears in system login logs, for example, and may have an impact on authentication, depending on which pam modules are in use.
+
+Support for this feature must be explicitly enabled via the new `ForwardedForHeader` option in the configuration file.
```

Comment:
Let's turn "configuration file" into a link to https://cockpit-project.org/guide/latest/cockpit.conf.5.html